aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -299,6 +332,9 @@ public AwsProperties(Map<String, String> properties) {
     this.glueCatalogId = properties.get(GLUE_CATALOG_ID);
     this.glueCatalogSkipArchive = PropertyUtil.propertyAsBoolean(properties,
         AwsProperties.GLUE_CATALOG_SKIP_ARCHIVE, AwsProperties.GLUE_CATALOG_SKIP_ARCHIVE_DEFAULT);
+    this.glueLakeFormationEnabled = PropertyUtil.propertyAsBoolean(properties,
```

Comment:
[minor] I think we should also initialize the default value of glueLakeFormationEnabled 
in the constructor above #L319

Reply:
done. thanks.